config/main.py

```diff
@@ -164,57 +164,32 @@ def _validate_interface_mode(ctx, BREAKOUT_CFG_FILE, interface_name, target_brko
 def load_configMgmt(verbose):
     """ Load config for the commands which are capable of change in config DB. """
     try:
-        cm = configMgmt(debug=verbose)
+        # TODO: set allowExtraTables to False, i.e we should have yang models for
+        # each table in Config. [TODO: Create Yang model for each Table]
+        # cm = configMgmt(debug=verbose, allowExtraTables=False)
+        cm = configMgmt(debug=verbose, allowExtraTables=True)
         return cm
     except Exception as e:
-        raise Exception("Failed to load the config. Error: {}".str(e))
+        raise Exception("Failed to load the config. Error: {}".format(str(e)))
 
+def breakout_Ports(cm, delPorts=list(), addPorts=list(), portJson=dict(), \
+    force=False, loadDefConfig=True, verbose=False):
 
-def delete_Ports(cm, final_delPorts,force_remove_dependencies, verbose):
-    """
-    Delete all ports.
-    del_ports: list of port names.
-    force_remove_dependencies: if false return dependecies, else delete dependencies.
-    Return:
-        WithOut Force: (xpath of dependecies, False) or (None, True)
-        With Force: (xpath of dependecies, False) or (None, True)
-    """
-    deps, ret = cm.deletePorts(delPorts=final_delPorts, force=force_remove_dependencies)
-    # No further action with no force and deps exist
-    if not force_remove_dependencies:
-        if ret == False and deps:
+    deps, ret = cm.breakOutPort(delPorts=delPorts, addPorts = addPorts, \
+                    portJson=portJson, force=force, loadDefConfig=loadDefConfig)
+    # check if DPB failed
+    if ret == False:
+        if not force and deps:
             print("Dependecies Exist. No further action will be taken")
-            print("\n*** Printing dependecies ***\n {}".format(deps))
+            print("*** Printing dependecies ***")
```

Comment:
Can we open a ticket to print the redis-DB data here as well just to be more customer friendly?

Reply:
I will open a ticket. But not sure How easy that will be to determine for each node. Though printing Table and primary key should be relatively easy.

Reply:
For time being I will simply return TABLE and Primary Key name, because those are easy to find. I will open a ticket for that.